data_steward/cdr_cleaner/cleaning_rules/clean_height_weight.py

```diff
@@ -590,113 +601,180 @@
-        new_weight_rows=NEW_WEIGHT_ROWS)
-    queries.append(weight_rows_query)
-
-    delete_weights_query = dict()
-    delete_weights_query[cdr_consts.QUERY] = DELETE_WEIGHT_ROWS_QUERY.format(
-        project_id=project_id, dataset_id=dataset_id)
-    queries.append(delete_weights_query)
-
-    insert_weights_query = dict()
-    insert_weights_query[cdr_consts.QUERY] = INSERT_NEW_ROWS_QUERY.format(
-        project_id=project_id,
-        dataset_id=dataset_id,
-        sandbox_dataset_id=sandbox_dataset_id,
-        new_rows=NEW_WEIGHT_ROWS)
-    queries.append(insert_weights_query)
-
-    return queries
-
-
-def parse_args():
+    Normalizes all height and weight data into cm and kg
+    and removes invalid/implausible data points (rows)
     """
-    Add sandbox_dataset_id to the default cdr_cleaner.args_parser argument list
 
-    :return: an expanded argument list object
-    """
-    import cdr_cleaner.args_parser as parser
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initializes the class with the proper information.
+
+        Set the issue numbers, description and affected datasets. As other tickets
+        may affect this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+        desc = (f'Normalizes all height and weight data into cm and kg '
+                f'and removes invalid/implausible data points')
+        super().__init__(issue_numbers=ISSUE_NUMBERS,
+                         description=desc,
+                         affected_datasets=[cdr_consts.DEID_BASE],
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         affected_tables=[CONCEPT, PERSON, 'measurement_ext', MEASUREMENT,
+                                          CONDITION_OCCURRENCE, CONCEPT_ANCESTOR])
+
+    def get_query_specs(self):
+        """
+        Returns a list of dictionary query specifications.
 
-    additional_argument = {
-        parser.SHORT_ARGUMENT: '-n',
-        parser.LONG_ARGUMENT: '--sandbox_dataset_id',
-        parser.ACTION: 'store',
-        parser.DEST: 'sandbox_dataset_id',
-        parser.HELP: 'Please specify the sandbox_dataset_id',
-        parser.REQUIRED: True
-    }
-    args = parser.default_parse_args([additional_argument])
-    return args
+        :return: A list of dictionaries. Each dictionary contains a single query
+            and a specification for how to execute that query. The specifications
+            are optional but the query is required.
+        """
+
+        # height
+        save_height_table_query = {
+            cdr_consts.QUERY:
+                CREATE_HEIGHT_SANDBOX_QUERY.render(
+                    project_id=self.get_project_id(),
+                    dataset_id=self.get_dataset_id(),
+                    sandbox_dataset_id=self.get_sandbox_dataset_id(),
```

Comment:
You need to pull develop and rebase.  These will be properties now (more pythonic) instead of getters and setters (more java like).